packages/core/src/login/oidc/IClientRegistrar.ts

```diff
@@ -58,6 +58,17 @@ function isValidUrl(url: string): boolean {
   }
 }
 
+export function determineSigningAlg(
+  supported: string[],
+  preferred: string[]
+): string | null {
+  return (
+    preferred.find((signingAlg) => {
+      return supported.includes(signingAlg);
+    }) || null
```

Comment:
Theoretically shouldn't cause any problems here, but generally good practice to avoid casting if unnecessary.

```suggestion
    }) ?? null
```

Reply:
77abf75ad79c767e18d6515ff269cafdbb260c0e